### ISSUE TITLE
Crash 3.90 go to Favorites / Tracks

### ISSUE THREAD

**Issue by attilagib** [osmandapp/OsmAnd-iOS#1108]
In the last version (3.90.4), on my IPhone 11, this release became unusable because the application crashes everything times we go in the Favorites/Traces.

**Comment by attilagib** [comment]
After reinstall of the app, the issue dispeared.

**Comment by R3gi** [comment]
I confirm the same issue on iPhone XR with iOS 14.4. OsmAnd crashes when I try to access My Places > Trips screen.

**Comment by EugeneZmeuk** [comment]
I have discovered a serious program bug. If I create a track and I rename it in my native language (Greek) the program crashes. Following the renaming in local language, the opening: My Places -> Tracks immediately crashes osmand maps app. Everytime. From there on, it is impossible to enter Tracks, access them and rename them back to English. Only deleting and reinstalling the app (with loss of user data) solves the problem... The problem is consistent. I have replicated over and over...
iPhone 11 Pro Max, iOS 14.4 (18D52).
